CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -1226,14 +1241,42 @@ if nameAndHealthMenuItem and topbarEnabled and not isTenFootInterface then
 end
 
 local Panel3D = require(GuiRoot.Modules.Panel3D)
+
 local function MoveHamburgerTo3D()
 	LeftMenubar:RemoveItem(settingsIcon)
 
 	local settingsIcon3D = Create3DSettingsIcon(TopBar)
 
+	local function OnHamburger3DInput(actionName, state, inputObj)
+		if state ~= Enum.UserInputState.Begin then
+			return
+		end
+		settingsIcon3D:Toggle()
```

Comment:
Since many different keys can now activate this toggle we might want to call a Set instead of toggle so the state doesn't get messed up